### ISSUE TITLE
Videos of flock2 or tello_ros?

### ISSUE THREAD

**Issue by Luke-A-F** [clydemcqueen/flock2#41]
I am considering using this for research. Are there any videos on Youtube or something showing off the implementation of this? Thanks.

**Comment by clydemcqueen** [comment]
@Luke-A-F thanks for the interest.

tl;dr there are no videos showing the implementation.

flock2 works well in simulation, but we had mixed results controlling the Tello drones in the real world. The drones seem to take commands as suggestions, and act sluggishly, and so the PID controllers tend to overreact even when the terms are set very loosely.

I've had several thoughts:

1. there are 2 APIs, the published API and the unpublished API that's used by the Tello app. tello_ros uses the published API. It's possible that the unpublished API is more responsive. Take a look at the TelloPy project for inspiration here.

2. the Tello drones have an on board IMU and down-facing camera that I can't access (the API for the IMU is not useful) but these sensors are clearly influencing flight in ways that I'm not modeling. With luck the low-level API will be responsive enough that it won't matter.

I haven't worked on this project in a while, but I'm always up for ideas and contributions!
